src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -110,6 +112,16 @@ public static Event from(Instant startTimestamp) {
                 .build();
     }
 
+    public Event bookName(String bookName) {
+        if (isNotBlank(bookName)) {
+            if (this.bookName != null) {
```

Comment:
The default value of this.book is DEFAULT_BOOK_NAME than this check will be false always.
Let's use the DEFAULT_BOOK_NAME value in the method which converted to proto.

Reply:
As we discussed I removed default book name